src/pe_reports/stylesheet.py

```diff
@@ -5,6 +5,13 @@
 from pptx.enum.chart import XL_LEGEND_POSITION
 from pptx.util import Pt
 
+# Font Attributes
+SMALL = Pt(10)
+LARGE = Pt(28)
+BLUE = RGBColor(3, 37, 126)
+GREEN = RGBColor(20, 200, 100)
+WHITE = RGBColor(255, 255, 255)
+
 
 class Paragraph:
```

Comment:
Should these methods be static? They do not appear to use `self` at all.

Reply:
@mcdonnnj  I had some back and forth with @jsf9k on this issue. - https://github.com/cisagov/pe-reports/pull/6/files/faf20b39ebfee1b21a034c622d448c483e4a427f#r651176744
I guess the usage is that the values creating the colors like the font sizes "Small", "Large" will never change and considered a value. Like a css element that aligns with brand standards. Scope may change on a full report buildout, but if it is a pythonic issue, I'd look for any suggestions to resolve.

Reply:
> shape

@mcdonnnj I am not sure I am seeing the issue clearly? Can you elaborate where the problem is showing.
Thanks...

Reply:
Sorry, what I'm referring to specifically is that these methods do need to be in an object to track state. They are just convenience utility methods to get desired formatting. As an example

```python
def text_style_title(self, font):
        """Text style for cover page title."""
        font.name = "Calibri"
        font.size, font.color = LARGE, WHITE
        return font
```

would instead be written as

```python
@staticmethod
def text_style_title(font):
        """Text style for cover page title."""
        font.name = "Calibri"
        font.size, font.color = LARGE, WHITE
        return font
```

Reply:
Resolved in f04e778